src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_utils.py

```diff
@@ -34,20 +34,24 @@ def _arm_get_resource_by_name(cli_ctx, resource_name, resource_type):
     :param str resource_type: The type of resource
     """
     result = get_resources_in_subscription(cli_ctx, resource_type)
-    elements = [item for item in result if item.name.lower() == resource_name.lower()]
+    elements = [item for item in result if item.name.lower() ==
+                resource_name.lower()]
```

Comment:
nit: this seems to be unnecessary extra work since the original line is also within the limit, but no need to revert either...